### PR TITLE
Expanded and clarified the GCP setup.

### LINKED ISSUE #533: Getting started on GCP error: DM failed because not enough quota
Someone at the summit reported that it's really hard to get started using Kubeflow on GCP.
- they got a quota error on DM. New project probably doesn't have enough cpu quota? 
- didn't know how to spend the 300 gcp credit
- so many [steps](https://www.kubeflow.org/docs/started/getting-started-gke/) to get started. Do we have doc for basic auth @kunmingg ?

### PR DESCRIPTION
**Pull request by sarahmaddox**
Fixes #533

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/556)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-556--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-gke/

**Comment by sarahmaddox** [comment]
This PR is ready for review.
Adding a hold to allow for multiple reviewers

/hold

**Comment by lluunn** [comment]
/lgtm
/approve
thanks!

File: content/docs/started/getting-started-gke.md [hunk 1]
```diff
@@ -30,14 +30,36 @@ brings the following advantages:
...
+1. Ensure that the following APIs are enabled on your GCP
+  account:
 
   * [Compute Engine](https://console.cloud.google.com/apis/library/compute.googleapis.com)
   * [GKE](https://console.cloud.google.com/apis/library/container.googleapis.com)
   * [Identity and Access Management (IAM)](https://console.cloud.google.com/apis/library/iam.googleapis.com)
   * [Deployment Manager](https://console.cloud.google.com/apis/library/deploymentmanager.googleapis.com)
 
+1. Check to see if you are eligible for the 
+  [GCP Free Tier](https://cloud.google.com/free/docs/gcp-free-tier), which gives
```
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/started/getting-started-gke.md]: Done.

File: content/docs/started/getting-started-gke.md [hunk 2]
```diff
@@ -30,14 +30,36 @@ brings the following advantages:
...
+(GCP) account:
+
+1. Select or create a project on the 
+  [GCP Console](https://console.cloud.google.com/cloud-resource-manager).
+
+1. Make sure that billing is enabled for your project. See the guide to
+  [modifying a project's billing 
+  settings](https://cloud.google.com/billing/docs/how-to/modify-project).
+
+1. Ensure that the following APIs are enabled on your GCP
```
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/started/getting-started-gke.md]: Done.

**Comment by sarahmaddox** [comment]
@lluunn PTAL and re-LGTM

**Pull request merged**